jakartaee10/JakartaEE10ReleasePlan.md

```diff
@@ -4,3 +4,167 @@
 
 Current state can be found at this Google doc:
 https://docs.google.com/document/d/1U24VmTzAfXcn3WBnVcolb8vhZO-Pnk_bit0CKh_d2jM/edit?usp=sharing
+
+### Preamble:  
+_Jakarta EE 10 is the first major release of Jakarta EE since the “jakarta” namespace update.
+Many of the component Specifications are introducing Major or Minor version updates to signify key changes in their respective Specifications and APIs.
+The Platform and/or Web Profile Specifications will be incorporating many of these updated Specifications._
+
+## Scope ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/352))
+The goal of the Jakarta EE 10 release is to deliver a set of coordinated specifications across the spectrum of Jakarta EE technologies. 
+
+The Platform team sees Jakarta EE 10 as the collection of component Specifications.
+Some of these component Specifications are introducing breaking changes to their API, which will require an increase in the Major version of the respective Specification.
+Many other component Specifications are introducing updates which are binary compatible and, thus, will only require a Minor version update.
+Regardless of the scope of changes, all artifacts for the component and Platform Specifications will be affected -- Specifications, APIs, TCKs, and Compatible Implementations.
+
+### Core Profile ([issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/387))
+Complementing the Web Profile, a new profile called the Core Profile is being proposed for Jakarta EE 10.
+The goal of the Core Profile is to define a subset of the Jakarta EE technologies which target microservices development and runtimes.
+It is still not clear if the creation of the Core Profile will be ready for Jakarta EE 10.
```

Comment:
I would say it is not clear if the Core Profile will be released in sync with the Web and Platform specs.

Reply:
Thanks.  Better explanation.  I'll adjust.